apps/web/src/views/Profile/components/EditProfileModal/StartView.tsx

```diff
@@ -114,7 +118,7 @@ const StartPage: React.FC<React.PropsWithChildren<StartPageProps>> = ({ goToAppr
                 onConfirm={needsApproval === true ? goToApprove : goToChange}
                 confirmLabel={t('Change Profile Pic')}
               />
-            </Flex>
+            </ChangeProfilePicWrapper>
```

Comment:
Don't think that we need the border bottom line

Reply:
Fixed